gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopSyncableOutputStreamIntegrationTest.java

```diff
@@ -88,7 +88,7 @@ public void hsync() throws Exception {
     assertThat(gcsFsIHelper.readFile(path)).isEqualTo(expected);
 
     TestUtils.verifyDurationMetric(
-        stats, GhfsStatistic.STREAM_WRITE_CLOSE_OPERATIONS, expected.length);
+        stats, GhfsStatistic.STREAM_WRITE_CLOSE_OPERATIONS, expected.length + 1);
```

Comment:
why?

Reply:
One write for each hsync() and one for the actual file. This was masked due to the bug in the test verification code.

Reply:
mind adding a comment here?